.config/nginx/dev/nginx.conf

```diff
@@ -25,6 +25,9 @@
 #     }
 # }
 
+# http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cache_path
+proxy_cache_path /var/cache/nginx levels=1:2 keys_zone=STATIC_CACHE:10m inactive=7d use_temp_path=off max_size=10g;
```

Comment:
@alashow, should we add a Docker volume to the `dev` service (in docker-compose.yml) so that the cache persists when containers are removed and recreated? (We would also have to figure out something for production environment, though, since production Nginx is in an LXC container.)

Reply:
@alashow, maybe we could create another task for this in Jira.  (It could be lower priority than some other things you are working on.)